src/App.js

```diff
@@ -1,7 +1,29 @@
 import React from 'react';
+import { Route, Link, Routes } from 'react-router-dom';
 import Calculator from './Components/calculator';
+import Home from './home';
+import './App.css';
+import Quote from './quote';
 
 function App() {
-  return <Calculator />;
+  return (
+    <>
+      <nav>
+        <ul>
+          <li><Link to="/home">Home</Link></li>
+          <div className="verticalLine" />
+          <li><Link to="/calculator">Calculator</Link></li>
+          <div className="verticalLine" />
+          <li><Link to="/quote">Quote</Link></li>
+        </ul>
+      </nav>
+      <Routes>
+        <Route path="/home" element={<Home />} />
+        <Route path="/calculator" element={<Calculator />} />
+        <Route path="/quote" element={<Quote />} />
+      </Routes>
+    </>
```

Comment:
- [ ] The home component should display on the root route `/` to match the wireframe. The home component is supposed to display on the root route. If users open your app they have to click on the home button to go to the home page. But it should open on the root route by default. Please fix this issue.
  - You can consider the below code.
```ruby
<li><Link to="/">Home</Link></li>
.........
.........
<Route path="/" element={<Home />} />

Reply:
solved, thank you.